libs/dh/metering-point/feature-overview/src/lib/dh-metering-point.component.ts

```diff
@@ -144,5 +154,26 @@ export class DhMeteringPointComponent {
   meteringPoint = computed(() => this.meteringPointDetails()?.metadata);
   isEnergySupplierResponsible = computed(() => this.meteringPointDetails()?.isEnergySupplier);
 
+  url = toSignal(this.route.url);
+
+  constructor() {
+    effect(() => {
+      this.breadcrumbService.addBreadcrumb({
+        label: translate('meteringPoint.breadcrumb'),
+        url: getPath('metering-point'),
+      });
+      this.breadcrumbService.addBreadcrumb({
+        label: this.meteringPointId(),
+        url: this.router
+          .createUrlTree([
+            getPath<BasePaths>('metering-point'),
+            this.meteringPointId(),
+            getPath<MeteringPointSubPaths>('master-data'),
+          ])
+          .toString(),
+      });
```

Comment:
issue: navigating to `/measurements` clears the breadcrumbs. Going back to `/master-data` does not display them again.

Reply:
should be fixed